packages/modules/auth/src/services/index.ts

```diff
@@ -20,5 +21,10 @@ export const getServices = (): Array<Provider> => {
             deps: [ModuleConfigToken],
             scope: Scope.Singleton,
         },
+        {
+            useClass: LoginService,
+            provide: LoginService,
+            scope: Scope.Singleton,
+        },
```

Comment:
@tvhees I'd expect this update in combination as the ones in packages/modules/bigcommerce/src/services/index.ts and packages/modules/bigcommerce/src/services/login-service.ts to allow overriding classes.